components/applications-service/pkg/storage/postgres/db.go

```diff
@@ -96,6 +96,7 @@ type service struct {
 	DeploymentID int32  `db:"deployment_id"`
 	SupID        int32  `db:"sup_id"`
 	Channel      string `db:"channel"`
+	FullPkgIdent string `db:"full_pkg_id"`
```

Comment:
Can we match the Habitat terminology here? I would suggest changing this to `package_ident`.

From the [habitat docs](https://www.habitat.sh/docs/reference/#package_identifier):
> package_identifier
A Habitat package identifier, split apart into its constituent components

Reply:
@kmacgugan I saw your 👍which led me to believe you agree on the suggestion but I didn't see any change? 🤔